lib/commentOnOpenedPullRequest.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+var Cesium = require('cesium');
+var Promise = require('bluebird');
+var requestPromise = require('request-promise');
+
+var dateLog = require('./dateLog');
+var checkStatus = require('./checkStatus');
+
+var Check = Cesium.Check;
+var defined = Cesium.defined;
+
+module.exports = commentOnOpenedPullRequest;
+
+/** Post comment on newly opened pull request
+ *
+ * @param {Object} jsonResponse Full response from GitHub event
+ * @param {Object} headers Headers to use in GET requests
+ * @param {String} changesSignature Message to post if CHANGES.md has changed in PR
+ * @param {String} thirdPartySignature Message to post if any third party files have changed
+ * @param {String[]} thirdPartyFolders Folder paths to search for third party file changes
+ * @return {Promise<http.IncomingMessage | String>} Response of POST
+ * @throws {DeveloperError} if jsonResponse or headers are not objects
+ */
+function commentOnOpenedPullRequest(jsonResponse, headers, changesSignature, thirdPartySignature, thirdPartyFolders) {
+    Check.typeOf.object('jsonResponse', jsonResponse);
+    Check.typeOf.object('headers', headers);
+    var pullRequest = jsonResponse.pull_request;
+    if (!defined(pullRequest)) {
+        return Promise.reject('jsonResponse has no key `pull_request`: ' + JSON.stringify(jsonResponse));
```

Comment:
Should this be a `DeveloperError`?  This will never happen because of a bad message from GitHub, will it?